cmd/algorand-indexer/daemon.go

```diff
@@ -4,8 +4,10 @@ import (
 	"context"
 	"fmt"
 	"os"
+	"strings"
 	"time"
 
+	"github.com/prometheus/client_golang/prometheus"
```

Comment:
a little confusing that this is different than what's used in api/server.go

Reply:
I changed it so that the echo module uses a different package name. This way only the prometheus client code will use that package.